draft-opsarea-rfc5706bis.md

```diff
@@ -97,29 +104,31 @@ author:
    The operational environment and manageability of the protocol should
    be considered from the start when new protocols are designed.
 
-   Most of the existing IETF management standards are focused on using
-   Structure of Management Information (SMI)-based data models (MIB
-   modules) to monitor and manage networking devices.  As the Internet
-   has grown, IETF protocols have addressed a constantly growing set of
+   As the Internet has grown, IETF protocols have addressed a constantly growing set of
    needs, such as web servers, collaboration services, and applications.
    The number of IETF management technologies has been expanding and the
    IETF management strategy has been changing to address the emerging
-   management requirements.  The discussion of emerging sets of
-   management requirements has a long history in the IETF.  The set of
-   management protocols you should use depends on what you are managing.
+   management requirements. In the past, most of the existing IETF management
+   standards were focused on using Structure of Management Information (SMI)-based
+   data models (MIB modules) to monitor and manage networking devices.
+   These days, the YANG data modeling language {?RFC7950}} is recommended to
+   monitor and manage the IETF protocols and the networking devices.
+   The discussion of emerging sets of management requirements has a long
```

Comment:
This reads odd.  Maybe, management requirements have evolved over the history of the IETF?